src/br/com/vaasschool/model/Question.java

```diff
@@ -1,25 +1,31 @@
 package br.com.vaasschool.model;
 
+import br.com.vaasschool.model.validation.Validator;
+
 public class Question extends Activity{
 
-    private String utterance;
+    private String statiment;
```

Comment:
Acho que seria assim:

```suggestion
    private String statement;
```

Aí, tem que corrigir o construtor, o getter e o setter.